src/controllers/module/event/eventController.ts

```diff
@@ -44,106 +45,135 @@ export class EventController {
 
 	/* istanbul ignore next */
 	private setRouterPaths() {
-		this.router.param('courseId', async (req, res, next, courseId) => {
-			const course = await this.learningCatalogue.getCourse(courseId)
+		this.router.param(
```

Comment:
Agree we probably don't need the function. We should change the module/event params function to use `req.params.courseId` instead of `res.locals.course.id`.

Reply:
I agree that it's not necessary in the controller, but course.id and module.id are used on every page for paths